WebRoot/public/extuserprov.jsp

```diff
@@ -67,7 +67,11 @@
 				<table>
 					<tr>
 						<td><app:img id="ZLoginErrorIcon" altkey='ALT_ERROR' src="dwt/ImgCritical_32.png" /></td>
-						<td id="errorMessage"></td>
+						<td id="errorMessage">
+							<% if (request.getAttribute("errorCode") != null) { %>
+								<fmt:message bundle="${zhmsg}" key='${errorCode}'/>
```

Comment:
NO_SHARE_EXISTS was added. But you can also get AccountServiceException.INVALID_PASSWORD
In case of any unknown exception you may get ServiceException.FAILURE or TEMPORARILY_UNAVAILABLE